fortio/stage/Makefile

```diff
@@ -2,22 +2,30 @@
 # TODO: parametrize the parent directory yaml etc instead of copy pasta
 # on the other hand it is safer to not accidentally damage the prod version.
 
+
+FORTIO_VERSION:=latest
+FORTIO_IMAGE:=istio/fortio:$(FORTIO_VERSION)
+FORTIO_MAX_STREAMS:=0
+
 deploy-fortio:
-	$(ISTIO_DIR)/bin/istioctl kube-inject --debug=false -f fortio.yaml | kubectl apply -f -
+	sed -e s%FORTIO_IMAGE%$(FORTIO_IMAGE)%g -e s%FORTIO_MAX_STREAMS%$(FORTIO_MAX_STREAMS)%g fortio.yaml | $(ISTIO_DIR)/bin/istioctl kube-inject --debug=false -f - | kubectl apply -f -
```

Comment:
You could also use [envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html), e.g.

```yaml

# Makefile
export FORTIO_VERSION = latest
export FORTIO_IMAGE = istio/fortio:$(FORTIO_VERSION)
export FORTIO_MAX_STREAMS = 0

deploy-fortio:
    envsubst < fortio.yaml | $(ISTIO_DIR)/bin/istioctl kube-inject --debug=false -f - | kubectl apply -f -

# fortio/stage/fortio.yaml
spec:
  containers:	       containers:
  - name: fortio-debug
    image: ${FORTIO_IMAGE}
    ports:	         ports:
    - containerPort: 8079 # grpc echo
    - containerPort: 8080 # http echo/debug
    args:
    - server
    - -grpc-max-streams
    - ${FORTIO_MAX_STREAMS}
```

Reply:
I keep forgetting about envsubst, will give it a shot, ty! 
(will keep this open for a bit)